### DISCUSSION TITLE
Are the new Nova and Whisper models available for the streaming (websocket) API?

### DISCUSSION THREAD

**Discussion by davehorton** [deepgram/community#118] [General help] [answered]
Docs are just not clear on this...

**Comment by SandraRodgers** [comment, marked as answer]
Hi @davehorton ,

Thanks for bringing that up. We'll update the docs with that information.

The Nova model is available with streaming. The code snippets in [this Quickstart](https://developers.deepgram.com/documentation/getting-started/streaming/) include `model=nova`, so you can follow this guide to try it out with the Python or Node SDK. If you're using another SDK, just add `model=nova` to use our newest most powerful model.

Deepgram's Whisper Cloud is not available with streaming.

Let us know if you need any help getting started.

Sandra

Edit: Here is a link to all the features and which model supports them: https://developers.deepgram.com/documentation/features/

**Comment by davehorton** [comment]
Hmm, here is the path I am using to test out nova and the deepgram server rejects the connection
```
/v1/listen?tier=general&model=nova&language=en-US&encoding=linear16&sample_rate=8000
```
What is wrong?  (I am not using python, I am using my own websocket client built in C)

**Reply by rastern** [reply]
Hi Dave. Try removing "tier=general" from your parameters.

**Comment by davehorton** [comment]
hmm, even with conversationalai it is taking 3 secs to connect to deepgram and no transcriptions coming back.  Is there a system issue today?

**Reply by SandraRodgers** [reply]
Are you still having problems connecting?

**Reply by iamnafets** [reply]
I'm having the same issue. Sometimes I get responses, sometimes I don't. Non-deterministic.

**Comment by davehorton** [comment]
I will have to check later as I am away from my office at the moment.  Will update here once I try again.

**Reply by SandraRodgers** [reply]
Thank you. And we have updated our docs to make it more clear that streaming is not available on Whisper (but is on Nova). We appreciate you calling our attention to that!

**Comment by davehorton** [comment]
Yes, this is working now.
